schema-loader/src/main/java/com/scalar/db/schemaloader/command/SchemaLoaderCommand.java

```diff
@@ -71,12 +72,22 @@
       defaultValue = "false")
   private boolean deleteTables;
 
+  @Option(
+      names = {"--prefix"},
+      description = "Namespace prefix",
+      hidden = true)
+  private String prefix;
```

Comment:
Please remove it.

Reply:
Sorry, but if we remove this then the namespace prefix is applied only for the meta table and coordinator table?

Reply:
I think rather than adding prefix option, we should change the content of the schema file.

Reply:
If we do that, then the schema file sample must be modified each running time. I think we can just use prefix here, then we don't have to change the schema file dynamically.

Reply:
Please don't forget the prefix is only for test. So we basically don't want to change the main code only for it as much as possible (off course, it's okay if we really need it). If we have a way that's not changing the main code, we should do that unless the way is too expensive.

Reply:
I see the point, thank you. I updated the code to dynamically change the schema file [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/aa0a087c02f328990bf34f07275bbc3730461a9f)